pkg/volume/emptydir/empty_dir.go

```diff
@@ -331,10 +331,10 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 		return nil
 	}
 
-	var options []string
+	options := []string{"noswap"}
 	// Linux system default is 50% of capacity.
 	if ed.sizeLimit != nil && ed.sizeLimit.Value() > 0 {
-		options = []string{fmt.Sprintf("size=%d", ed.sizeLimit.Value())}
+		options = append(options, fmt.Sprintf("size=%d", ed.sizeLimit.Value()))
```

Comment:
Please add some unit tests verifying the noswap option.

I have a feeling this code will change but let's make sure to have some unit test coverage also.

Reply:
Done!